ansible/deploy.yml

```diff
@@ -24,8 +24,9 @@
           - unzip
       when: full_deploy
 
-- include: firewall.yml
+- import_playbook: firewall.yml
   when: full_deploy
-- include: docker.yml
+- import_playbook: docker.yml
   when: full_deploy
-- include: beryllium.yml
+- import_playbook: beryllium.yml
+- import_playbook: fail2ban.yml
```

Comment:
```suggestion
- import_playbook: fail2ban.yml
  when full_deploy
```

also move fail2ban.yml to just after the filrewall.yml

Reply:
@eoliveros can you make this change

Reply:
changed to the suggestion above

Reply:
doing deployment and seeing if the chnges are still there

Reply:
* also move fail2ban.yml to just after the filrewall.yml

I want beryllium to be the last thing installed and the security protocols to be the first installed

Reply:
moved the fail2ban.yml under firewall.yml